packages/react-storage/src/components/StorageImage/StorageImage.tsx

```diff
@@ -99,11 +100,16 @@ export const StorageImage = ({
     ]
   );
 
-  const { url } = useGetUrl(input);
+  const { url, isLoading } = useGetUrl(input);
+
+  if (isLoading) {
+    return <View style={{ ...style }}></View>;
+  }
```

Comment:
Rather than destructure `style`, let's add a `loadingElement` key on `StorageImageProps` and render it conditionally while `isLoading` is `true` if provided.

```ts
interface StorageImageProps {
  loadingElement?: React.ReactNode;
}
```